### PR TITLE
chore: added a project

### LINKED ISSUE #1480: chore: project addition by Illaf
### Add a new project to the list

Adding the UI of a cat purchasing website

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue
- [X] My project has contribution guidelines
- [X] My project has a Code of conduct
- [X] My project has README
- [X] My project has a License

### PR DESCRIPTION
**Pull request by Illaf**
## Related Issue

Closes #1480 

## Description

Added the UI design of a cat purchasing website

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -3643,5 +3643,23 @@
...
+  "Social_media": {
+    "gitHub": "https://github.com/Illaf",
+    "LinkedIn": "https://www.linkedin.com/in/elaf-shafeeq-764420243",
+    "Instagram": "#",
+    "YouTube": "#",
+    "Twitter": "#"
+  },
+  "Projects": [
+    {
+      "link": "https://illaf.github.io/The-Cat-Store/",
```
Comment (priyankarpal) [inline DB/projects.json:3658]: add github link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Illaf** [comment]
@priyankarpal I have added the changes. Please merge now

**Review by priyankarpal — APPROVED** [review]

**Comment by Illaf** [comment]
@priyankarpal Please add a GSSOC tag in this PR . I am  a GSSOC contributor. It is necessaary for me!

**Comment by priyankarpal** [comment]
> @priyankarpal Please add a GSSOC tag in this PR . I am a GSSOC contributor. It is necessaary for me!

previously we've added but currently, We are not providing any labels for project additions as they are not associated with solving any issues.

**Pull request merged**